gio/Cargo.toml

```diff
@@ -13,7 +13,7 @@ exclude = [
     "gir-files/*",
 ]
 edition = "2021"
-rust-version = "1.57"
```

Comment:
This already happened in https://github.com/gtk-rs/gtk-rs-core/commit/86eeafe6e0b1d4133df2c5d5f8c1fdd5edff7e24 or not?

Reply:
Yes, I was in an ancient local copy, tanks.